src/app/components/views/login-view/login-view.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-6 offset-md-3">
+      <div class="card">
+        <div class="card-body text-center">
+          <h5 class="card-title ">ABC Local Logon</h5>
+          <p class="card-text">Log in using your super secret ABC ID that is not at all made up</p>
+          <form (ngSubmit)="onLogin()" [formGroup]="uidForm">
+            <div class="form-group">
+              <label for="uid">ID:</label>
+              <input type="text" id="uid" class="form-control" formControlName="uid">
+              <label for="password">Password:</label>
+              <input type="password" class="form-control">
+            </div>
+             <button type="submit" class="btn btn-success" [disabled]="(!uidForm?.valid)">Login!</button>
```

Comment:
I think the form looks good, could just use some space around the button.
<img width="338" alt="Screen Shot 2023-04-30 at 12 36 26 AM" src="https://user-images.githubusercontent.com/11700686/235335714-44357ecd-c59f-4da4-a132-510205593a1a.png">

Reply:
agree, i'll take a look at it tomorrow morning